Queueomatic/Client/Components/Profile/ProfileComponent.razor

```diff
@@ -0,0 +1,101 @@
+﻿@using Blazored.SessionStorage
+@inject HttpClient HttpClient;
+@inject ISessionStorageService SessionStorageService;
+@inject NavigationManager NavigationManager;
+
+<div class="padding">
+    <div class="container d-flex justify-content-center">
+        <div class="card user-card-full">
+            <div class="row">
+                <div class="col-sm-4 bg-c-lite-green user-profile">
+                    <div class="card-block text-center text-white">
+                        <div class="m-b-25">
+                            @* img src "https://icons8.com/icon/pm4OSDf9HTVG/class-dojo" *@
+                            <img src="avatar.png" class="img-radius" alt="User-Profile-Image">
+                        </div>
+                        <h6 class="f-w-600">@User.NickName</h6>
+                        <p>A Nice Person</p>
+                        <i class=" mdi mdi-square-edit-outline feather icon-edit m-t-10 f-16"></i>
+                    </div>
+                </div>
+                <div class="col-sm-8">
+                    <div class="card-block">
+                        <h6 class="m-b-20 p-b-5 b-b-default f-w-600">
+                            Information
+                            @* TODO: Uncomment this section to activate the edit button*@
```

Comment:
I am not sure what these comments do. When uncommenting everything I get errors. Should these not be removed?

Reply:
This button triggers the `dialog` element down in the page so make sure to uncomment that to.